### PR TITLE
fix: add Tejender1521

### LINKED ISSUE #67: Send custom html to the GitHub Action plugin

### LINKED ISSUE #1683: [PROFILE ADDITION] Tejender Upadhyay
### Profile screenshot

Screenshot from 2023-01-24 22-07-13

### PR DESCRIPTION
**Pull request by Tejender1521**
resolves #1683

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Tejender1521.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/75739873/214353017-4b8112af-e9f6-4ead-8d73-0a0a7224b1ee.png",
+  "issueId": 67,
```
Comment (Panquesito7) [inline profiles/Tejender1521.json:3]: Wrong issue ID.
```suggestion
  "issueId": 1683,
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by himanshu1221 — APPROVED** [review]
Looks Nice to me !

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**